.github/dependabot.yml

```diff
@@ -44,3 +44,14 @@ updates:
     commit-message:
       prefix: "deps"
       include: "scope"
+  - package-ecosystem: "gomod"
+    directory: "/updater"
+    schedule:
+      interval: "weekly"
+    ignore:
+      # Opentelemetry updates will be done manually
+      - dependency-name: "github.com/open-telemetry/opentelemetry-collector*"
+      - dependency-name: "go.opentelemetry.io/collector/*"
```

Comment:
We likely won't need these but we can keep them for now.

Reply:
Yeah, I was thinking we PROBABLY won't need these, but it shouldn't hurt.